atlasdb-dbkvs-hikari/src/test/java/com/palantir/nexus/db/pool/config/OracleConnectionConfigTest.java

```diff
@@ -72,19 +73,32 @@ public void databaseUrlGeneratedCorrectlyFromServiceName() {
     }
 
     @Test
-    public void namespaceIsSidIfPresent() {
+    public void namespaceDefaultsToSidIfPresent() {
         OracleConnectionConfig connectionConfig = getBaseBuilder().sid(SID).build();
         assertThat(connectionConfig.namespace()).contains(SID);
     }
 
     @Test
-    public void namespaceIsNamespaceOverrideIfServiceNameConfigurationSpecified() {
+    public void namespaceDefaultsToNamespaceOverrideIfServiceNameConfigurationSpecified() {
         OracleConnectionConfig connectionConfig = getBaseBuilder()
                 .serviceNameConfiguration(SERVICE_NAME_CONFIGURATION)
                 .build();
         assertThat(connectionConfig.namespace()).contains(SERVICE_NAME_CONFIGURATION.namespaceOverride());
     }
 
+    @Test
```

Comment:
Please can we have a test to show that `databaseUrlGeneratedCorrectlyFromNamespaceOverride`?

Perhaps also an ETE test that uses the namespace override, especially if that is to be the expected behaviour from now on.